models/cam-fv/model_mod.f90

```diff
@@ -136,6 +136,9 @@ module model_mod
           init_time,                     &
           init_conditions
 
+! public routine for utility program
+public :: obs_too_high
+
```

Comment:
The obs_too_high routine is in cam-common-code. 
Rather than making obs_too_high public from cam-fv, can we have  cam_dart_obs_preprocessor
use cam_common_code::obs_too_high.
That way the cam_dart_obs_preprocessor can be used for cam-fv and cam-se?

@nancycollins @kdraeder  is obs_too_high intended to be used with both cam-fv and cam-se?

Reply:
Thanks Nancy!  
Looking at the common code module, it looks like the vertical stuff is in there.  We'll bug Kevin when he gets back.

Reply:
I think for now it is probably best to get this released for cam-fv. 
If it is super useful, then in the future look at adding this obs_too_high program so it can be used with any model.
@mjs2369 let me know what you think.

Reply:
Let's leave the public :: obs_too_high in cam-fv.